src/views/bouquets/BouquetFormView.vue

```diff
@@ -122,6 +126,16 @@ const cancel = () => {
   }
 }
 
+const metaTitle = (): string => {
+  return topic.value.name
+    ? `Éditer le bouquet ${topic.value.name}`
```

Comment:
Utiliser la conf ici et plus bas pour "bouquet".

Reply:
Je m'en suis rendu compte, j'ai pas encore poussé !

Reply:
Oups, j'attendrai ton "re-request review" la prochaine fois !

Reply:
J'essaie de couvrir le cas de figure dans lequel une page n'aurait ni de `meta.title` ni de `useHead` pour envoyer un warning mais c'est plus compliqué que prévu…

Reply:
Ne t'embête pas trop, c'était un nice to have puisque c'était facile à faire, mais ce n'est plus le cas.